src/components/navbar.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Button, FormControl, MenuItem, Select, styled } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+
+export default function Navbar() {
+  const { t, i18n } = useTranslation()
+  const [language, setLanguage] = React.useState('en')
+  const changeLanguage = (lang: string) => {
+    i18n.changeLanguage(lang)
+    setLanguage(lang)
+  }
+
+  const NavbarCont = styled('div')`
```

Comment:
styled.div``

Reply:
tutaj w takim przypadku mam error TS `TS2339: Property 'div' does not exist on type 'CreateMUIStyled '`

Reply:
jak cos to jest z `import { styled } from '@mui/material'`

Reply:
to chyba tak zostawimy bo tak jest w dokumentacji : https://mui.com/system/styled/